python/foglamp/plugins/south/cc2650async/cc2650async.py

```diff
@@ -348,6 +348,9 @@ def plugin_reconfigure(handle, new_config):
         new_handle = plugin_init(new_config)
         new_handle['restart'] = 'yes'
         _LOGGER.info("Restarting CC2650ASYN plugin due to change in configuration keys [{}]".format(', '.join(diff)))
+    elif 'pollInterval' in diff or 'connectionTimeout' in diff or 'shutdownThreshold' in diff:
+        new_handle = copy.deepcopy(new_config)
+        new_handle['restart'] = 'no'
```

Comment:
what diff these lines will make; does not else do the same thing? and also what is `pollInterval` doing here?

Reply:
copy.deepcopy() is being called with "new_config" in elif and with "handle" in else.